### PR TITLE
Business dashboard theme - circular progress bar adjustments

### PR DESCRIPTION
**Pull request by dimafeldman**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/Themes/businessDashboard/components/CircularProgressBar.st.css [hunk 1]
```diff
@@ -33,15 +54,22 @@
...
-  top: calc(50% - 6px);
+}
+
+.root::progressBar:size(medium) {
+  height: 56px;
+  width: 56px;
+}
+
+.root::progressBar:size(medium)::arcsContainer {
+  height: 56px !important;
```
Comment (dimafeldman) [inline src/Themes/businessDashboard/components/CircularProgressBar.st.css:65]: I couldn't find a "cleaner" way to override the progress bar size :(

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6179

File: src/Themes/businessDashboard/components/CircularProgressBar.st.css (hunk 1 above)
Comment (shlomitc) [reply src/Themes/businessDashboard/components/CircularProgressBar.st.css:65]: Yes, that's quite nasty.
The reason behind it is that the `wix-ui-core` component uses inline style to apply a `size`.
To make it css only will require a change both in the core component and in the WSR component that uses it.

You can keep it that way but let's just add a comment here like:
`/* TODO - requires a change in wix-ui-core to remove inline sizes */ `

File: src/Themes/businessDashboard/docs/components/CircularProgressBar.js [hunk 2]
```diff
@@ -5,7 +5,8 @@ export const playground = `
     <CircularProgressBar showProgressIndication size="small" value={50} />
```
Comment (shlomitc) [inline src/Themes/businessDashboard/docs/components/CircularProgressBar.js:5]: IIRC, I think that you only need the medium size, so maybe we can just remove this one from the examples and from the CSS. Can you please verify that only the medium size is needed?

Comment (dimafeldman) [reply src/Themes/businessDashboard/docs/components/CircularProgressBar.js:5]: yes, we will use only the medium size. will remove it

**Pull request merged**